### PR TITLE
Update poly_example to use more accurate observations

### LINKED ISSUE #3150: Set ENKF_TRUNCATION in poly_ert
Set ENKF_TRUNCATION in poly_ert and update documentation.

### PR DESCRIPTION
**Pull request by dafeda**
A lot of uncertainty in observations makes it
difficult to see the effect that the update step has
on each parameter.
It also makes it hard to see the effect that a truncation value of 0.98 has.
By reducing the errors in observations, it is easy to see that a truncation value of 1.00
is necessary if we want the update step to improve all three parameters.
ENKF_TRUNCATION is therefore set to 1.00 in the config.

**Issue**
Resolves #3150 

## Pre review checklist

- [x ] Added appropriate labels

Adding labels helps the maintainers when writing release notes, see sections and the
corresponding labels here: https://github.com/equinor/ert/blob/main/.github/release.yml

### CODE REVIEW

**Comment by BjarneHerland** [comment]
The failure you see here [`Python package / Run ert tests (3.6, ubuntu-latest, unit-tests) (pull_request)`](https://github.com/equinor/ert/runs/5665752475?check_suite_focus=true)
is a suspected/known flaky - please rebase to include https://github.com/equinor/ert/commit/32b4350c0694418d12042905c1af6da189827000 and see if you get it again.

**Comment by dafeda** [comment]
> The failure you see here [`Python package / Run ert tests (3.6, ubuntu-latest, unit-tests) (pull_request)`](https://github.com/equinor/ert/runs/5665752475?check_suite_focus=true) is a suspected/known flaky - please rebase to include [32b4350](https://github.com/equinor/ert/commit/32b4350c0694418d12042905c1af6da189827000) and see if you get it again.

Will do, thanks.

File: test-data/local/poly_example/time_map [hunk 1]
```diff
@@ -1 +1 @@
-1/10/2006
+01/10/2006
```
Comment (berland) [inline test-data/local/poly_example/time_map:1]: Think you can use `2006-10-01` now.

File: docs/rst/manual/getting_started/configuration/poly_new/guide.rst [hunk 2]
```diff
@@ -399,19 +397,19 @@ The observations need to relate to some results of the simulation, so that the
...
 were measured at the points 0, 2, 4, 6 and 8. The indices here happen to align
 with the x values of the polynomial evaluation, but this is incidental. The
-indices says where the observations in the file match the result indices. Put
+indices say where the observations in the file match the result indices. Put
 the following observations in the file ``poly_obs_data.txt``:
 
     .. literalinclude:: with_observations/poly_obs_data.txt
 
-The observations are written one for each line, with the first number
+There is one observations per line, with the first number
```
Comment (berland) [inline docs/rst/manual/getting_started/configuration/poly_new/guide.rst:405]: observations -> observation

File: docs/rst/manual/getting_started/configuration/poly_new/guide.rst [hunk 3]
```diff
@@ -399,19 +397,19 @@ The observations need to relate to some results of the simulation, so that the
...
 signifying the observed value, and the second number signifying the uncertainty.
 
 Now let's describe the observations we have to ERT. This is done with the
```
Comment (berland) [inline docs/rst/manual/getting_started/configuration/poly_new/guide.rst:408]: "we have to ERT", another typo that you can fix :). ERT is not yet a verb, maybe later..

File: docs/rst/manual/getting_started/configuration/poly_new/guide.rst [hunk 4]
```diff
@@ -440,10 +438,22 @@ these lines::
 The :ref:`OBS_CONFIG <obs_config>` line simply tells ERT that there is a
 description of an observation set in the file ``observations``. The
 :ref:`TIME_MAP <time_map>` is legacy, and not used anymore, but it is still
-required to create a `time_map` file (e.g. containing 00/00/0000) when 
+required to create a `time_map` file (e.g. containing 01/06/2006) when
```
Comment (berland) [inline docs/rst/manual/getting_started/configuration/poly_new/guide.rst:441]: ISO-dates, 2006-06-01

File: test-data/local/poly_example/time_map (hunk 1 above)
Comment (dafeda) [reply test-data/local/poly_example/time_map:1]: Thanks, it seems to work.

File: docs/rst/manual/getting_started/configuration/poly_new/guide.rst (hunk 3 above)
Comment (dafeda) [reply docs/rst/manual/getting_started/configuration/poly_new/guide.rst:408]: ERT is the subject here :)
We are describing the observations to ERT - ERT is to whom we are describing the observations! 
But yeah, I'll try and come up with a better sentence.

Comment (dafeda) [reply docs/rst/manual/getting_started/configuration/poly_new/guide.rst]: I've pushed a new proposal.

File: docs/rst/manual/getting_started/configuration/poly_new/with_observations/poly_obs_data.txt [hunk 5]
```diff
@@ -1,5 +1,5 @@
-2.8532509308 0.1
-7.20311703432 1.1
-21.3864899107 4.1
-31.5145559347 9.1
-53.5676660405 16.1
+2.857617496354537 0.1
```
Comment (ManInFez) [inline docs/rst/manual/getting_started/configuration/poly_new/with_observations/poly_obs_data.txt:1]: make error scale with the observed value maybe?

**Comment by dafeda** [comment]
I've updated the observations such that their errors are increasing with the value of the polynomial, as suggested by @ManInFez .

The coefficient `b` does not get updated as much now, which is due to few data-points with quite a bit of noise.
The new estimates are however better, so I think we are good.

**Comment by ManInFez** [comment]
Needs a rebase on main to get style test to go through

**Review by ManInFez — APPROVED** [review]
👍

**Pull request merged**